test/chplcheck/PREDIFF

```diff
@@ -5,5 +5,5 @@ $CHPL_HOME/tools/chplcheck/chplcheck --enable-rule ConsecutiveDecls \
   --enable-rule UnusedFormal \
   --enable-rule CamelOrPascalCaseVariables \
   $1.chpl >> $2
-sed -i .tmp "s#$(pwd)/##" $2 # strip the working directory from output
+sed -i.tmp "s#$(pwd)/##" $2 # strip the working directory from output
```

Comment:
`sed -i` is pretty unportable anyways, I'd suggest just changing to something more portable in general and just make a tmp file. 

for example
```
mv $2 $2.tmp
cat "s#$(pwd)/##" $2.tmp >$2
rm $2.tmp
```